octobot_trading/channels/orders.py

```diff
@@ -113,11 +113,14 @@ async def update_order_from_exchange(self, order,
         :param force_job_execution: When True, order_update_job will bypass its dependencies check
         :return: True if the order was updated
         """
-        await get_chan(ORDERS_CHANNEL, self.channel.exchange_manager.id).producers[-1].\
-            update_order_from_exchange(order=order,
-                                       should_notify=should_notify,
-                                       force_job_execution=force_job_execution,
-                                       wait_for_refresh=wait_for_refresh)
+        try:
+            await get_chan(ORDERS_CHANNEL, self.channel.exchange_manager.id).producers[-1]. \
+                update_order_from_exchange(order=order,
+                                           should_notify=should_notify,
+                                           force_job_execution=force_job_execution,
+                                           wait_for_refresh=wait_for_refresh)
+        except IndexError:
+            self.logger.warning("Can't synchronize order from exchange")
```

Comment:
Should this be an error ? 
I think if we really need to update an order (if a websocket didn't receive an order update for ex), we should be able to update it via the rest interface.

Reply:
I don't know how we can do that with the current architecture